src/pages/Registration/Registration.tsx

```diff
@@ -1,27 +1,62 @@
 import React from "react";
 
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 
 import FormBuilder from "src/components/FormBuilder/FormBuilder";
 import FormWrapper from "src/components/FormWrapper/FormWrapper";
+import { Layout } from "src/components/Layout/Layout";
+import { PageRoutes } from "src/constants/routeNames";
+
+import { LocalStorageKey } from "src/enums/localStorageEnum";
+
+import { UseActions } from "src/hooks/useActions";
+import { useAppSelector } from "src/hooks/useAppSelector";
+
+import { FormBuilderReturnType } from "src/interfaces/formbuilder-returnvalue";
+import { UserInput } from "src/types/UserInput";
 
 import config from "./config";
 
 import "./style.scss";
 
 const Registration = () => {
+  const navigate = useNavigate();
+  const user = useAppSelector((state) => state.user);
+  console.log(user);
```

Comment:
remove console.log
Please check in all places

Reply:
Removed.